src/components/AppBar/AppBar.jsx

```diff
@@ -43,14 +44,18 @@ const styles = (theme) => ({
   },
 });
 
-const AppBar = ({
-  isLoggedIn, selected, classes, routes,
-}) => {
+// eslint-disable-next-line
+const AppBar = ({ isLoggedIn, selected, classes, routes }) => {
```

Comment:
```suggestion
const AppBar = ({
  isLoggedIn,
  selected,
  classes,
  routes,
}) => {
```
Then you won't need the eslint disable anymore

Reply:
thanks!